streamx-console/streamx-console-service/src/main/java/com/streamxhub/streamx/console/core/task/CheckpointProcessor.java

```diff
@@ -51,27 +53,29 @@ public void process(Long appId, CheckPoints checkPoints) {
         CheckPoints.Latest latest = checkPoints.getLatest();
         if (latest != null) {
             Application application = applicationService.getById(appId);
+            String jobId = application.getJobId();
             CheckPoints.CheckPoint checkPoint = latest.getCompleted();
```

Comment:
you can:

String cacheId = appId + "_" + jobId;

checkPointCache.get(cacheId);
...
checkPointCache.put(cacheId, checkPoint.getId());

Reply:
ok, i change this code